x-pack/legacy/plugins/license_management/public/np_ready/application/sections/license_dashboard/license_dashboard.js

```diff
@@ -25,7 +27,7 @@ export const LicenseDashboard = ({ setBreadcrumb } = { setBreadcrumb: () => {} }
         <EuiFlexItem>
           <AddLicense />
         </EuiFlexItem>
-        <StartTrial />
+        <StartTrial telemetry={npStart.plugins.telemetry} shouldShowTelemetryOptIn={true} />
```

Comment:
`shouldShowTelemetryOptIn={true}` is not expected by `StartTrial`. Did you mean `shouldShowStartTrial`?